bin/n3h

```diff
@@ -30,8 +30,14 @@ async function main () {
     dir: logDir
   })
 
-  // setup logging
-  tweetlog.set('t') // enable trace logging
+  // Setup logging
+  // Default to trace level
+  tweetlog.set('t')
+  // Check log level setting in env
+  const hasLogLevel = 'N3H_LOG_LEVEL' in process.env
+  if (hasLogLevel) {
+    tweetlog.set(process.env.N3H_LOG_LEVEL)
+  }
```

Comment:
I'm cool with this for now.

Just to stick it in your brain for the future... we've now got two different configuration paths, one through environment vars like this, and one that is piped into the process via stdin on spawn. I feel like we may want to move all configuration into the stdin pipe for consistency.

Reply:
... after looking at the rust side of this, maybe it actually makes more sense for this to be a env variable... since otherwise we'd have to inspect / inject it into the otherwise opaque networking config...

Reply:
Adding it to RTB. Do you think this should be done soon?

Reply:
huh? I wrote my initial comment before seeing your second one.

The way I currently see it, we can keep both:
 - stdin pipe is for the opaque networking config, which I would rather call the end-user network config.
- env vars are for the developer network config (set by the hApp/conductor/Holo developer)